integration_test.go

```diff
@@ -1947,10 +1947,6 @@ func TestMachinePolicyResource(t *testing.T) {
 			t.Fatal("The machine policy must have a ConnectionRetryTimeLimit of \"00:05:00\" (was \"" + fmt.Sprint(resource.ConnectionRetryTimeLimit) + "\")")
 		}
 
-		if resource.PollingRequestMaximumMessageProcessingTimeout.Minutes() != 10 {
-			t.Fatal("The machine policy must have a PollingRequestMaximumMessageProcessingTimeout of \"00:10:00\" (was \"" + fmt.Sprint(resource.PollingRequestMaximumMessageProcessingTimeout) + "\")")
-		}
```

Comment:
Do we have this validation in the API also?

Reply:
Property was deprecated in server in 2024.2